docs/language-lead-handbook.md

```diff
@@ -99,10 +60,23 @@ article5title: 'Cosa è API?'
 article5link: 'https://www.freecodecamp.org/italian/news/cose-un-api-in-italiano-per-favore/'
 ```
 
-You can convert from one format to the other carefully changing it manually. Or you can use [the script in this repl](https://replit.com/@Ieahleen/convert-json-to-yaml).
+Each number represents one of the 30 articles in the footer. Make sure to match the title and the link correctly.
 
-> [!TIP]
-> A new workflow is being worked on, there will be only one place to change in the future.
+For each article, you will need to create a shorter title to use in the footer. Each title must stay on a single line and not go to a new line.
+
+You will want to [build the translated client locally](how-to-enable-new-languages.md) to see if the titles have the right length. You can preview the changes by editing the `trending.json` file in your local. Here are the detailed steps:
+
+1. Update your `.env` file to use your language for `CLIENT_LOCALE` and `CURRICULUM_LOCALE`.
+
+2. Run `pnpm run develop`. This will automatically generate `trending.json` file for your language under `/client/i18n/locales/` directory.
+
+3. Stop the running command with `Ctrl+C` in the terminal.
```

Comment:
```suggestion
2. Run `pnpm run create:config`. This will automatically generate the `trending.json` file for your language under the `/client/i18n/locales/` directory.
```

Should be able to call the create command directly.